addons/mail/static/src/core/common/message.js

```diff
@@ -432,6 +432,7 @@ export class Message extends Component {
         this.props.messageEdition?.exitEditMode();
         message.composer = undefined;
         this.state.isEditing = false;
+        message.isEditing = false;
```

Comment:
Note: goal is to remove state in favor of model field but we can't do this in stable.